### PR TITLE
IndexOptionsConflict for chunk collection - pr

### PR DESCRIPTION
**Pull request by sekko27**
#364

### CODE REVIEW

File: tests/cases/06_gridfs.ts [hunk 1]
```diff
@@ -35,14 +35,15 @@ testWithClient("GridFS: Echo large Image", async (client) => {
...
+      upstream,
+    );
 
   const getId =
     (await bucket.find({ filename: "deno_logo.png" }).toArray())[0]._id;
 
-  const expected = await fetch("https://deno.land/images/deno_logo.png");
+  const expected = await fetch("https://deno.land/images/artwork/snow_den.png");
   const actual = await new Response(await bucket.openDownloadStream(getId))
     .arrayBuffer();
```
Comment (erfanium) [inline tests/cases/06_gridfs.ts:49]: Please either revert these changes, because we already localized these assets so this PR has merge conflicts

**Review by erfanium — CHANGES_REQUESTED** [review]

File: tests/cases/06_gridfs.ts [hunk 2]
```diff
@@ -113,3 +114,21 @@ testWithClient(
...
+      const bucket = new GridFSBucket(client.database("test"), {
+        bucketName: "sameKeys",
+      });
+      const upstream = await bucket.openUploadStream(`test-asset-${index}`);
+      const writer = upstream.getWriter();
+      await writer.write(new TextEncoder().encode(`[asset${index}]`));
+      await writer.close();
+    };
+    await addAsset(0);
+    await addAsset(1);
```
Comment (lucsoft) [inline tests/cases/06_gridfs.ts:132]: Where is the assert in this code? Shouldn’t the test case somehow check if the indexes where skipped?

**Review by lucsoft — CHANGES_REQUESTED** [review]

**Review by lucsoft — APPROVED** [review]
:shipit: 

LGTM
